user-journey-tests/specs/identification/taskListVerification.spec.js

```diff
@@ -0,0 +1,35 @@
+import {
+  loginAndSaveSession,
+  restoreSession
+} from '../../helpers/authSessionManager'
+import { enableIdentification } from '../../helpers/testHelpers/flow-helpers/enableIdentification.js'
+import taskListPage from '../../page-objects/taskListPage'
+import signInPage from '../../page-objects/signInPage.js'
+
+describe('Identification - task list appearance', () => {
+  // eslint-disable-next-line no-undef
+  before(async () => {
+    await loginAndSaveSession(signInPage)
+    await taskListPage.navigateToPageAndVerifyTitle()
+  })
+
+  beforeEach(async () => [await restoreSession()])
```

Comment:
square brackets here should be curlies: 

```javascript
beforeEach(async () => { await restoreSession() })
```

To be fair, even that is redundant - the `beforeEach` function is expecting a function that takes no arguments & returns a promise (aka a nullary async function) - which is what restoreSession already is.

So we can do:

```javascript
beforeEach(restoreSession)
```